### DISCUSSION TITLE
Set limits and requests for Longhorn related resources?

### DISCUSSION THREAD

**Discussion by fmunteanu** [longhorn/longhorn#9697] [Q&A]
I'm deploying Longhorn with the Helm chart and I cannot find any settings where I could define the resource limits and requests, for example `engine-image-ei`, `longhorn-manager`, `node-driver-registrar`, `longhorn-liveness-probe`, etc. Is there a way to define them?

**Comment by DamiaSan** [comment]
I don't think so, with Helm you can set resource and limits only after the installation.

**Comment by fmunteanu** [comment]
This was requested in the past but closed without any resolution: https://github.com/longhorn/longhorn/issues/1502

Resource requests and limits should be defined into Helm chart options, this is how is done everywhere.

**Comment by DamiaSan** [comment]
I understand your needs Floren, but as you can see in the issue you posted this topic has already been discussed in the past, so there isn't much that we can do here.

**Comment by derekbit** [comment]
@fmunteanu We have an ticket https://github.com/longhorn/longhorn/issues/8332 for addressing the resource constraint. You can follow up it
